Makefile

```diff
@@ -1,15 +1,9 @@
-TESTS  ?= --enable-tests
```

Comment:
Please do not overwrite the non-generated Makefile. The previous Makefile added various hooks which are necessary.